### PR TITLE
expression: rewrite builtin function: RADIANS

### PR DESCRIPTION
**Pull request by zz-jason**
to https://github.com/pingcap/tidb/issues/4080

### CODE REVIEW

File: expression/builtin_math.go [hunk 1]
```diff
@@ -1384,31 +1384,26 @@ func (c *radiansFunctionClass) getFunction(args []Expression, ctx context.Contex
...
-	if arg.IsNull() {
-		return d, nil
-	}
-	degree, err := arg.ToFloat64(b.ctx.GetSessionVars().StmtCtx)
-	if err != nil {
-		return d, errors.Trace(err)
+func (b *builtinRadiansSig) evalReal(row []types.Datum) (float64, bool, error) {
+	x, isNull, err := b.args[0].EvalReal(row, b.ctx.GetSessionVars().StmtCtx)
+	if isNull || err != nil {
+		return 0, true, errors.Trace(err)
```
Comment (winkyao) [inline expression/builtin_math.go]: `return 0, isNull, errors.Trace(err)`, isNull may be false.

File: expression/builtin_math_test.go [hunk 2]
```diff
@@ -721,6 +721,8 @@ func (s *testEvaluatorSuite) TestRadians(c *C) {
 		fc := funcs[ast.Radians]
 		f, err := fc.getFunction(datumsToConstants(t["Arg"]), s.ctx)
 		c.Assert(err, IsNil)
+		c.Assert(f, NotNil)
+		c.Assert(f.isDeterministic(), IsTrue)
```
Comment (winkyao) [inline expression/builtin_math_test.go]: why check `isDeterministic` here ? you can check it once in the end .

**Comment by XuHuaiyu** [comment]
LGTM

**Comment by shenli** [comment]
LGTM

**Pull request merged**